RELEASE/scripts/autoscend/auto_quest_level_12.ash

```diff
@@ -267,169 +279,106 @@ int auto_warPlan()
 	int advCostLighthouse = 10;	//placeholder estimate. TODO actual math
 	int advCostOrchard = 10;	//placeholder estimate. TODO actual math
 	int advCostNuns = 20;		//placeholder estimate. TODO actual math
```

Comment:
I guess future work would be to create auto_estimatedAdventuresForLighthouse/Orchard/Nuns.
I don't know how you'd do the maths for the junkyard as it's so RNG based.

Reply:
Agreed, a future PR should refactor the nuns logic to split off turn estimation.

Reply:
also agreed. we need to move the nuns estimation code from its own function (which is an utter mess to read) into a dedicated calculation function. and then replace this fixed 20 value with a call to said function.
I plan on working on it in the future after i am done with my current PRs.